pages/singlesite/[slug].js

```diff
@@ -34,17 +34,36 @@ export default function singleSite() {
           clickable: true,
         }}
         modules={[Pagination]}
-        onSlideChange={() => console.log("slide change")}
-        onSwiper={(swiper) => console.log(swiper)}
       >
         <StyledSwiperSlide>
-          <StyledImage
-            className="document"
-            src={currentSite.document}
-            alt={currentSite.adress}
-            height={200}
-            width={300}
-          />
+          <Swiper
+            className="mySwiper2 swiper-v"
+            direction={"vertical"}
+            spaceBetween={0}
+            slidesPerView={1}
+            pagination={{
+              clickable: true,
+            }}
+            modules={[Pagination]}
+          >
+            <StyledSwiperSlide>
+              <div className="info-card">
+                <h2>
+                  {currentSite.coordinates[0] +
+                    " | " +
+                    currentSite.coordinates[1]}{" "}
+                </h2>
+                <h2>{currentSite.adress}</h2>
+                <p className="info">{currentSite.information[0]}</p>
+              </div>
+            </StyledSwiperSlide>
+
+            <StyledSwiperSlide>
```

Comment:
do you always have 2 information entires?  I suggest to add some logic to make this dynamic ;)